webodf/lib/ops/OdtDocument.js

```diff
@@ -938,6 +979,48 @@ ops.OdtDocument = function OdtDocument(odfCanvas) {
     };
 
     /**
+     * Process steps being inserted into the document. Will emit a steps inserted signal on
+     * behalf of the caller
+     * @param {!{position: !number}} args
+     * @return {undefined}
+     */
+    this.handleStepsInserted = function(args) {
+        stepsTranslator.handleStepsInserted(args);
+        // signal not used in webodf, but 3rd-party (NVivo)
+        eventNotifier.emit(ops.OdtDocument.signalStepsInserted, args);
+    };
+
+    /**
+     * Process steps being removed from the document. Will emit a steps removed signal on
+     * behalf of the caller
+     * @param {!{position: !number}} args
+     * @return {undefined}
+     */
+    this.handleStepsRemoved = function(args) {
+        stepsTranslator.handleStepsRemoved(args);
+        // signal not used in webodf, but 3rd-party (NVivo)
+        eventNotifier.emit(ops.OdtDocument.signalStepsRemoved, args);
+    };
+
+    /**
+     * Emit the signal that the passed cursor moved.
+     * @param {!ops.OdtCursor} cursor
+     * @return {undefined}
+     */
+    this.emitSignalCursorMoved = function(cursor) {
+        eventNotifier.emit(ops.Document.signalCursorMoved, cursor);
```

Comment:
Annnd yet another update. So, movedCursor doesn't emit (oops... rusty brain). fixCursorPositions definitely should not be emitting directly however.